lib/fluent/plugin/out_mongo.rb

```diff
@@ -213,23 +213,30 @@ def collect_records(chunk)
         if date_keys
           date_keys.each { |date_key|
             begin
-              date_value = record[date_key]
+              path = date_key.split('.')
```

Comment:
How about to use record_accessor instead of using hand-written implementation?
ref: https://docs.fluentd.org/plugin-helper-overview/api-plugin-helper-record_accessor

Reply:
Hi, 
I have implemented the solution you have proposed (using record_accessor).
Let me know if something is wrong or I need to fix something.

Thank you